actions/user_actions.jsx

```diff
@@ -173,7 +173,7 @@ export function loadNewDMIfNeeded(channelId) {
     };
 }
 
-export async function loadNewGMIfNeeded(channelId) {
+export function loadNewGMIfNeeded(channelId) {
```

Comment:
This `async` wasn't actually being used, but ended up tripping a test once we started dispatching this method. Fixed to align with `loadNewDMIfNeeded` which wasn't `async`.